package.json

```diff
@@ -95,7 +95,8 @@
     },
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
-    ]
+    ],
+    "roots": ["src"]
```

Comment:
Could you format `"src"` on a new line? Next time yarn/npm does something to this file, it'll reformat it and I'd prefer it to not change this line :)